src/common/entity/state_active.ts

```diff
@@ -24,7 +21,7 @@ export function stateActive(stateObj: HassEntity, state?: string): boolean {
     case "device_tracker":
     case "person":
       return compareState !== "not_home";
-    case "media-player":
+    case "media_player":
       return compareState !== "idle" && compareState !== "standby";
```

Comment:
@bramkragten Do you think we must consider `idle` state as active ? According to core documentation, the device is turned on but no media or unknown media (e.g. HDMI input for receiver ?) https://developers.home-assistant.io/docs/core/entity/media-player/#states)

Reply:
I think it depends on the device, for a receiver, probably yes? For a chromecast, probably no?

Reply:
May be we can use the device class :
idle active for `receiver`
idle inactive for `tv` and `speaker`

Reply:
After multiple discussion with users, it's better to consider it "active" as the device is turned on